jgrapht-core/src/test/java/org/jgrapht/alg/HopcroftKarpBipartiteMatchingTest.java

```diff
@@ -107,4 +107,38 @@ public void testEmptyMatching(){
 		HopcroftKarpBipartiteMatching<Integer,DefaultEdge> bm= new HopcroftKarpBipartiteMatching<>(graph, new HashSet<>(partition1), new HashSet<>(partition2));
 		assertEquals(Collections.EMPTY_SET, bm.getMatching());
 	}
+	
+	/**
+	 * Issue 233 instance
+	 */
+	public void testBipartiteMatchingIssue233()
+    {
+        UndirectedGraph<Integer, DefaultEdge> g = new SimpleGraph<Integer, DefaultEdge>(
+            DefaultEdge.class);
+
+        Set<Integer> left = new HashSet<Integer>();
+        Set<Integer> right = new HashSet<Integer>();
+
+        g.addVertex(0);
+        left.add(0);
+        g.addVertex(1);
+        left.add(1);
+        g.addVertex(2);
+        right.add(2);
+        g.addVertex(3);
+        right.add(3);
```

Comment:
Could you simplify the above to:
Graphs.addAllVertices(g, IntStream.rangeClosed(0,3));
Set<Integer> left = new HashSet<Integer>(Arrays.asList(0,1));
Set<Integer> right = new HashSet<Integer>(Arrays.asList(2,3));